itou/employee_record/management/commands/sanitize_employee_records.py

```diff
@@ -94,16 +92,15 @@ def _check_missed_notifications(self, dry_run):
                 # Unarchive the employee record so next time we don't miss the notification
                 if employee_record.status == Status.ARCHIVED:
                     employee_record.unarchive()
-            self.stdout.write(f" - {total_created} notification(s) created")
-        self.stdout.write(" - done!")
+            self.logger.info("%d notifications created", total_created)
```

Comment:
Là c'est vraiment de la curiosité malsaine car les deux valeurs devraient toujours être plus ou moins synchro, _normalement_
```suggestion
            self.logger.info("%d/%d missed employee records notifications created", total_created, len(employee_record_with_missing_notification))
```